docs/user-guide/vpcs.md

```diff
@@ -197,3 +198,50 @@ spec:
   - from: 1000
     to: 2999
 ```
+
+## Mode
+
+VPCs can operate in two modes: L2VNI and L3VNI. L2VNI is the default mode of
+operation and represents the conventional functionality. L3VNI is designed
+for switches that lack the hardware ASIC features that allow them to operate on L2VNI encapsulation.
+
+### L2VNI Mode
+
+This is the conventional multi-tenant network virtualization mode. It is the
+default option for VPCs.
+
+
+### L3VNI Mode
+
+In L3VNI mode, the switches are configured to exclusively route unicast traffic.
+This enables multi-tenancy inside of a fabric, even with switches of mixed
+capabilities. The [DS5000](../reference/profiles.md#celestica-ds5000) is an
+L3-only leaf and VPCs attached to this switch must be in L3VNI mode.
+
+Without broadcast traffic each end host needs to have a full /32 address for
+its address (e.g., `10.10.0.5/32`, not `10.10.0.5/24`). The host also
+needs to emit traffic containing its IP-to-MAC mapping before the network will be
+able to route traffic to it.
+
+The DHCP server included with the Fabric has been updated to support L3VNI
+mode. When a VPC is using the included DHCP server and is in L3VNI mode,
+the DHCP server will send a DHCP lease with a short duration, so that the DHCP client will immediately request a new
+lease. The DHCP renewal request is sufficient traffic for the network to gain
+knowledge of the host and redistribute the route via BGP. Subsequent lease requests
+will use the configured lease duration.
+
+If a user elects to use their own DHCP server or statically assign IP addresses. It
+is recommended that the user set the following sysctl values:
+```console
+net.ipv4.conf.default.arp_notify=1
+net.ipv4.conf.default.arp_announce=1
+```
+The output of `ip route` should be similar to the following output with the IP
+addresses that match your environment.
+
+```console
+user@server ~$ ip route
+default via 10.10.0.1 dev enp2s1 proto static
+10.10.0.0/24 via 10.10.0.1
```

Comment:
it should be (asuming enabled default route) and it would probably benefit from some comments:

```
root@server-5:/# ip r
default via 10.30.2.1 dev enp2s1.1002 proto dhcp src 10.30.2.4 metric 1024
10.30.2.1 dev enp2s1.1002 proto dhcp scope link src 10.30.2.4 metric 1024 # route for the VPC subnet gateway
```

if default route is disabled then instead of it you'll have 10.30.2.0/24 via 10.30.2.1